infra/launch.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python
+
+import argparse
+import getpass
+import subprocess
+
+from infra import deploy
+from google.cloud import storage
+
+
+# Oddly enough, there's no API to simply fetch the current gcloud configuration...
+def gcloud_config():
+    client = storage.Client()
+    return {
+        "project": client.project,
+    }
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--tpu", type=str, required=True)
+    parser.add_argument("--project", type=str, default=gcloud_config()["project"])
+    parser.add_argument("--zone", type=str, required=True)
+    parser.add_argument("--docker_repository", type=str, required=True)
+    parser.add_argument("--wandb_api_key", type=str, required=False)
```

Comment:
i aspire to use google's secrets and/or some simple lightweight thing (read a .env file or something?) but haven't gotten there. a minimum, might be nice to have a -e (I sometimes pass in libtpu flags, e.g.)

Reply:
> would be nice if we could put this in a .config in the levanter base dir or something 

Good call. I added `.config_tpu` YAML and pulled default values for all the args from there. It looks like:

```
env:
    WANDB_API_KEY: ...
    WANDB_ENTITY: ...
    WANDB_PROJECT: levanter
    HF_TOKEN: ...

docker_repository: levanter
zone: us-west4-a
tpu: test-spin-up-32
```